app/pathogen/arbovirus/visualizations/page.tsx

```diff
@@ -0,0 +1,49 @@
+"use client"
```

Comment:
I love the idea of this as a separate page so that
1. Our page doesn't get huge and laggy (I also think some kind of solution with a modal would be more of a headache than this)
2. Someone can link a zoomed in version of a graph to someone else (Also helpful for citations?)
3. We can throw some more stuff on the page which I think would be fun!

Honestly Wikipedia when you want to click on an image to view it fullscreen was my real inspo here and I really liked how it turned out! It doesn't add load time, you can check out the vercel deploy.

Reply:
Oooo I really like this especially for our analytics even to see which graphs are being used